runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePlugin.java

```diff
@@ -57,6 +57,11 @@ MetronomePluginConfiguration provideConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(MetronomePluginConfiguration.class);
 	}
+	@Override
+	protected void startUp()
+	{
+		tickCounter = 0;
```

Comment:
This is good. We should also reset `shouldTock`.
```suggestion
		tickCounter = 0;
		shouldTock = false;
```